plugins/community-plugins/mq2aapurchase.md

```diff
@@ -14,25 +14,25 @@ You can find the latest version of MQ2AAPurchase [here](https://macroquest.org/p
 
 ## Commands
 
-`/aapurchase help -- List commands, aka this`  
-`/aapurchase now -- start buying now`  
-`/aapurchase add \"AA Name\" rank -- M for max instead of a rank`  
-`/aapurchase bank # -- set the amount of points to bank`  
-`/aapurchase load -- Reload INI`
+`/aapurchase help` -- List commands, aka this<br>
```

Comment:
I would look at using the definition list format like we use for datatypes/tlos, or using a list. Using html blocks like this is an indicator that you're going against the grain.